### PR TITLE
Use a more legible YAML format

### PR DESCRIPTION
**Pull request by goetzc**
And also fix an Ansible warning

### CODE REVIEW

File: tasks/bootstrap-tasks.yml [hunk 1]
```diff
@@ -1,75 +1,88 @@
 ---
-- name: Make sure that the authorized_keys are present for the admin user
+- name: Make sure that the authorized_keys file is set up for the admin user
   authorized_key:
     user: "{{ admin_user }}"
-    key: "{{ item }}"
+    key:  "{{ item }}"
```
Comment (00dav00) [inline tasks/bootstrap-tasks.yml]: Extra space?

File: tasks/bootstrap-tasks.yml [hunk 2]
```diff
@@ -1,75 +1,88 @@
...
+    port: "{{ item }}"
+    proto: tcp
   with_items:
     - "{{ ports | default([22]) }}"
 
 - name: Open ports by IP
-  ufw: rule=allow port={{ item.value.port }}  proto=tcp src={{ item.value.ip }}
+  ufw:
+    rule: allow
+    src:  "{{ item.value.ip }}"
```
Comment (00dav00) [inline tasks/bootstrap-tasks.yml]: Extra space

File: tasks/set-authorized-keys.yml [hunk 3]
```diff
@@ -1,18 +1,29 @@
 ---
-# tasks Add or remove ssh keys
+# Task to update authorized SSH keys
```
Comment (00dav00) [inline tasks/set-authorized-keys.yml]: Is this comment necessary given that the task name is descriptive?

File: tasks/set-authorized-keys.yml [hunk 4]
```diff
@@ -1,18 +1,29 @@
...
 - name: Set up authorized_keys for the admin user
-  authorized_key: user="{{ admin_user }}" key="{{ item }}" state=present exclusive=yes
+  authorized_key:
+    user: "{{ admin_user }}"
+    key:  "{{ item }}"
```
Comment (00dav00) [inline tasks/set-authorized-keys.yml]: Extra space

File: tasks/set-authorized-keys.yml [hunk 5]
```diff
@@ -1,18 +1,29 @@
...
+    state: present
+    exclusive: yes
   with_file:
     - keys/administrators
 
 - name: Set up authorized_keys for the deploy user
-  authorized_key: user="{{ deploy_username }}" key="{{ item }}" state=present exclusive=yes
+  authorized_key:
+    user: "{{ deploy_username }}"
+    key:  "{{ item }}"
```
Comment (00dav00) [inline tasks/set-authorized-keys.yml]: Extra space

File: tasks/set-authorized-keys.yml [hunk 6]
```diff
@@ -1,18 +1,29 @@
...
   with_file:
     - keys/deploy_users
   when: deploy_username is defined
 
-- name: Set up authorized_keys for ci on deploy user
-  authorized_key: user={{ deploy_username }} key="{{ item }}" state=present
+- name: Set up authorized_keys for CI on the deploy user
+  authorized_key:
+    user: "{{ deploy_username }}"
+    key:  "{{ item }}"
```
Comment (00dav00) [inline tasks/set-authorized-keys.yml]: Extra space

File: tasks/bootstrap-tasks.yml (hunk 1 above)
Comment (goetzc) [inline tasks/bootstrap-tasks.yml]: Exactly that's the idea, make it more legible, here using alignments.

**Pull request merged**